buildmodule_test.go

```diff
@@ -283,10 +283,6 @@ func testBuildpack(t *testing.T, context spec.G, it spec.S) {
 			resolver libpak.DependencyResolver
 		)
 
-		it.Before(func() {
```

Comment:
I'm not sure this is safe. There are tests that depend on the arch to operate consistently on an M1 or x86 system. Did you run your tests on different arch systems?